### ISSUE TITLE
Pull Request

### ISSUE THREAD

**Issue by fisicorj** [jakubgarfield/Bonobo-Git-Server#812]
Bonobo should have a control before the commit. As you have in github the pull request so that the changes can be verified and only after approval release the change.

**Comment by latop2604** [comment]
Duplicate of https://github.com/jakubgarfield/Bonobo-Git-Server/issues/493

**Comment by fisicorj** [comment]
Sorry. He had not found it. I already read what was written. so they can close.
